misc-utils/lsfd.c

```diff
@@ -1603,6 +1603,8 @@ static void __attribute__((__noreturn__)) usage(void)
 	for (i = 0; i < ARRAY_SIZE(infos); i++)
 		fprintf(out, " %16s  %-10s%s\n", infos[i].name,
 			infos[i].json_type == SCOLS_JSON_STRING?  "<string>":
+			infos[i].json_type == SCOLS_JSON_ARRAY_STRING?  "<string>":
+			infos[i].json_type == SCOLS_JSON_ARRAY_STRING?  "<string>":
```

Comment:
Sorry, I took mistake. The second SCOLS_JSON_ARRAY_STRING should be SCOLS_JSON_ARRAY_NUMBER. I'll make a pull request for fixing this.